tests/system/testdata/endtoend/http.txtar

```diff
@@ -46,10 +46,11 @@ project:
         path: "/test/{a}/{b...}"
 
 -- main.star --
-def on_http(data, trigger, event):
+def on_http(data, trigger, event, conn):
```

Comment:
Where are all the valid argument names for handlers documented?
This is something users must know.